azurelinuxagent/common/osutil/default.py

```diff
@@ -289,10 +289,13 @@ def is_current_instance_id(self, id_that):
         '''
         Compare two instance IDs for equality, but allow that some IDs
         may have been persisted using the incorrect byte ordering.
+        Ensure conformity with RFC-4122 allowing for case insensitivity.
         '''
         id_this = self.get_instance_id()
-        return id_that == id_this or \
-            id_that == self._correct_instance_id(id_this)
+        logger.verbose("id_this: {0}".format(id_this))
```

Comment:
Are these 2 log statements leftovers from debugging? If they are meant to actually be logged, could we improve the message. Thanks

Reply:
I added those following the initial rejection claiming the fix wasn't working.  Yes, we could make the message better.  Let me know the results of the automation before I open this up again.

Reply:
OK, thanks